pyGHDL/requirements.txt

```diff
@@ -1,3 +1,6 @@
 pydecor>=2.0.1
-pyVHDLModel==0.10.4
+pyAttributes==2.1.0
```

Comment:
Shall we create a requirements file for the DOM which is separated from pyGHDL?
It's ok to install all of those packages by default, but some users might want the LSP only, so they can get along with fewer dependencies.
So: `requirements.dom.txt`, `requirements.lsp.txt` and `requirements.txt` (including both of the others).

Reply:
Hmmm or the other way around, a `requirements.txt` for the minimal stuff and the `requirements.xxx.txt` for the extended usecases. I would prefer the latter approach, because that's what we do e.g. in docs where re recursively source the main file and add additional packages for docs.

I'm not sure if `requirements.xxx.txt` will be understood by dependency checkers. we might need separate directories from them. But this makes it quite complex, so maybe you're approch is better as it provides a simple file at top for everyone and for user who complain we have specific files ...

Reply:
I'm thinking about providing a default file and requiring users to understand others if they want a minimal setup. Hence, I don't think the structure is very relevant, as long as we document it. Up to you!

Maybe?

- `requirements.min.txt`
- `requirements.dom.txt` or `dom/requirements.txt`
- `requirements.lsp.txt` or `lsp/requirements.txt`
- `requirements.txt` (importing the three before; this is the default)